api/src/main/java/org/openmrs/module/eptsreports/reporting/data/converter/VlResultClassificationConverter.java

```diff
@@ -0,0 +1,35 @@
+package org.openmrs.module.eptsreports.reporting.data.converter;
+
+import org.openmrs.module.reporting.data.converter.DataConverter;
+
+/**
+ * Converter to check the Viral Load is greater 1000 copies/ml and Return its related Classification
+ */
+public class VlResultClassificationConverter implements DataConverter {
+  @Override
+  public Object convert(Object o) {
+
+    if (o == null) {
+      return "N/A";
+    }
+
+    double valueInDoubleform = Double.parseDouble(o.toString());
+    int value = (int) valueInDoubleform;
```

Comment:
This verification is not needed! The VL result is an integer value and its not recorded as doubles.